### PR TITLE
Private/mmeeks/configbits

### PR DESCRIPTION
**Pull request by mmeeks**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: docker/from-packages/scripts/start-collabora-online.sh [hunk 1]
```diff
@@ -33,12 +33,22 @@ if test "${SAL_LOG-set}" == set; then
...
+if test -n "${username}"; then
+    perl -pi -e "s/<username (.*)>.*<\/username>/<username \1>${username}<\/username>/" /etc/loolwsd/loolwsd.xml
+fi
+if test -n "${password}"; then
+    perl -pi -e "s/<password (.*)>.*<\/password>/<password \1>${password}<\/password>/" /etc/loolwsd/loolwsd.xml
+fi
+if test -n "${server_name}"; then
+    perl -pi -e "s/<server_name (.*)>.*<\/server_name>/<server_name \1>${server_name}<\/server_name>/" /etc/loolwsd/loolwsd.xml
+fi
+if test -n "${allowed_languages}"; then
```
Comment (thebearon) [inline docker/from-packages/scripts/start-collabora-online.sh:49]: This should be `${dictionaries}`

**Review by thebearon — CHANGES_REQUESTED** [review]
Found one issue with it, apart from that it worked fine for me.

File: docker/from-packages/scripts/start-collabora-online.sh (hunk 1 above)
Comment (mmeeks) [reply docker/from-packages/scripts/start-collabora-online.sh:49]: great catch - brain fade: thanks ! =) will merge and back-port to co-6-4 too ..

**Comment by mmeeks** [comment]
Abandoning - pushed directly with the trivial fix.

**Pull request closed without merging**